scv-prototype-frontend/src/app/views/Private/Profile/DatesOfBirth/DatesOfBirth.jsx

```diff
@@ -29,11 +35,13 @@ const DatesOfBirth = ({ datesOfBirth }) => {
 };
 
 DatesOfBirth.propTypes = {
-  datesOfBirth: PropTypes.arrayOf(PropTypes.shape(DateOfBirth.propTypes))
+  person: PropTypes.objectOf(PropTypes.object),
+  programs: PropTypes.arrayOf(PropTypes.object)
 };
 
 DatesOfBirth.defaultProps = {
-  datesOfBirth: []
+  person: null,
+  programs: []
```

Comment:
If you're supplying a default value, then I assume that the property is required? So your `propTypes` should declare that.

Also, why specify `null` as the default value for person? Why not just leave it undefined?

Reply:
It gives me an error if I doesn't put any defaultProps.

Reply:
That's strange. It shouldn't. I'm going to grab your branch and take a look.